bon-macros/src/builder/builder_gen/mod.rs

```diff
@@ -715,12 +717,59 @@ impl BuilderGenCtx {
             }
         };
 
-        let setter_methods = setters
+        let setget_methods = setgets
             .into_iter()
-            .map(|(setter_methods, _)| setter_methods)
+            .map(|(setter_method, _, maybe_getter_method)| {
+                [setter_method, maybe_getter_method.unwrap_or_default()].concat()
+            })
             .concat();
 
-        Ok((setter_methods, items_for_rustdoc))
+        Ok((setget_methods, items_for_rustdoc))
+    }
+
+    fn getter_method(&self, member: &RegularMember) -> Option<TokenStream2> {
+        member.param_getter().map(|()| {
+            // for comments on how this getter formed, please look to setter
```

Comment:
The `setter_methods.rs` file is quite a big one, and if you wanted to comment on something I'd recommend just copying the comment here or using some shared method between setters and getters

Reply:
will remove comments